server/service/integration_enterprise_test.go

```diff
@@ -11871,6 +11878,19 @@ func (s *integrationEnterpriseTestSuite) TestBatchSetSoftwareInstallers() {
 	require.NotNil(t, packages[0].URL)
 	require.Nil(t, packages[0].TeamID)
 
+	// maintained app with no_check for sha
```

Comment:
Similar case here. We're basically end-to-end testing FMA adds for 1pw and Chrome, which takes a bit because it involves downloading those files. Maybe we want to do this, but maybe we want this test to run separately/in parallel.